.github/workflows/update-version.yaml

```diff
@@ -30,6 +33,27 @@ jobs:
         fi
         echo "Extracted the version number '${version_number}'."
         echo "::set-output name=yb_version::${version_number}"
+    - name: "Switch to correct branch"
+      id: extract-branch
+      run: |
+        rel=${{steps.extract-version.outputs.yb_version}}
+        a=( ${rel//./ } )
+        branch_name="${a[0]}.${a[1]}"
```

Comment:
Come to think of it, maybe we should do this calculation on dispatcher side and send a branch_name parameter? Then we won't need the is_master parameter, we just get teh branch name from yugabyte-db side.

Reply:
If we do this, lets call it `yb_release` or maybe `yb_release_train` or `release_train` rather than branch.